.github/workflows/aws-cdk.yml

```diff
@@ -296,22 +342,32 @@ jobs:
       - name: Bootstrap CDK environment
         run: |
           echo "🥾 Bootstrapping CDK environment..."
-          
+
           verbose=""
           if [ "${{ inputs.debug }}" = "true" ]; then
             verbose="--verbose"
           fi
-          
-          role_args=""
-          if [ -n "${{ secrets.cfn-execution-role }}" ]; then
-            role_args="--cloudformation-execution-policies ${{ secrets.cfn-execution-role }}"
+
+          BOOTSTRAP_CMD="${{ needs.prepare.outputs.cdk-bootstrap-cmd }}"
+
+          # Check if using custom command from config or default
+          if [ "$BOOTSTRAP_CMD" = "npx cdk bootstrap" ]; then
+            # Default command - add AWS-specific arguments
+            role_args=""
+            if [ -n "${{ secrets.cfn-execution-role }}" ]; then
+              role_args="--cloudformation-execution-policies ${{ secrets.cfn-execution-role }}"
+            fi
+
+            $BOOTSTRAP_CMD \
+              aws://$(aws sts get-caller-identity --query Account --output text)/${{ inputs.aws-region }} \
+              $role_args \
+              $verbose
+          else
+            # Custom command from cdk-config.yml - run as-is with extra args
```

Comment:
From my understanding in the summary, I thought we're moving away from using the cdk-config.yml file?